package.json

```diff
@@ -39,6 +39,9 @@
     "install:font:path-serif": "replace 'https://ui.vlaanderen.be/assets/fonts/serif/FlandersArtSerif' 'https://cdn.milieuinfo.be/vlaanderen-font/LATEST/font/flandersart/FlandersArtSerif' ../../src/style.css",
     "install:font:path-italic": "replace 'https://ui.vlaanderen.be/assets/fonts/italic/FlandersArtSans' 'https://cdn.milieuinfo.be/vlaanderen-font/LATEST/font/flandersart/FlandersArtSansItalic' ../../src/style.css",
     "install:font:vlaanderen-icon": "replace '/font/iconfont/vlaanderen-icon.woff' 'https://cdn.milieuinfo.be/vlaanderen-font/LATEST/font/iconfont/3.9.1/vlaanderen-icon.woff' ../../src/style.css",
+    "pretest": "node build test/single/example example --noCommit && node build test/multiple/example example --noCommit ",
```

Comment:
Niet super belangrijk, maar het is gebruikelijk om woorden in argumenten te scheiden met een dash ipv. camelcase te gebruiken.